03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,32 @@
+const Task = require('../models/Task')
+const getAllTasks = (req, res) => {
```

Comment:
You need to implement this method to retrieve the task from the MongoDB. See the example located in the final/controller directory to know how to implement this.